packages/styled-components/src/constants.js

```diff
@@ -10,7 +10,7 @@ export const SC_STREAM_ATTR = 'data-styled-streamed';
 export const IS_BROWSER = typeof window !== 'undefined' && 'HTMLElement' in window;
 
 export const DISABLE_SPEEDY =
-  (typeof SC_DISABLE_SPEEDY === 'boolean' && SC_DISABLE_SPEEDY) ||
```

Comment:
`!!SC_DISABLE_SPEEDY || process.env.NODE_ENV !== 'production'` would accomplish the same thing right?

Reply:
No, it would not as the global variable itself may be undefined in some instances :)